qirlib/src/lib.rs

```diff
@@ -3,8 +3,17 @@
 
```

Comment:
Including the README.md file here as a crate doc attribute may be helpful for onboarding to crates.io and docs.rs. The qdk_sim_experimental crate in qsharp-runtime does something similar to reuse content between the crate and repo docs.